src/services/link/link-model.js

```diff
@@ -19,7 +19,7 @@ const linkSchema = new Schema({
 		retries: 20, // number of retries on collision
 	},
 	data: { type: Object },
-	target: { type: String, required: true },
+	target: { type: String, required: true, index: true },
```

Comment:
would not set this, links are selected by _id

Reply:
founds this in the logs

Reply:
Yes some request go over target.